app/emoji/client/emojiPicker.js

```diff
@@ -281,8 +267,10 @@ Template.emojiPicker.onCreated(function() {
 	};
 
 	this.autorun(() => {
-		if (this.recentNeedsUpdate.get()) {
-			this.recentNeedsUpdate.set(false);
+		if (recentEmojisNeedsUpdate.get()) {
```

Comment:
🤔 could you help me to understand what this code does?

Reply:
legacy cold